contracts/yellow_button/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
This crate seems to have `Cargo.toml`, we can set `license` attribute as well

Reply:
done